prow/kube/prowjob.go

```diff
@@ -148,6 +148,9 @@ type DecorationConfig struct {
 	// GCSCredentialsSecret is the name of the Kubernetes secret
 	// that holds GCS push credentials
 	GCSCredentialsSecret string `json:"gcs_credentials_secret,omitempty"`
+	// SshKeySecrets are the names of Kubernetes secrets that contain
+	// SSK keys which should be used during the cloning process
+	SshKeySecrets []string `json:"ssh_key_secrets,omitempty"`
```

Comment:
This effectively also allows for one top-level default for this field. That means if you put any keys in the default, all clone operations get them. However, since the secrets are only mounted into the container that seems safe to do. Or just opt-in when you need it and eat the verbosity cost.

Reply:
SGTM